jagannath_pandit.md

```diff
@@ -0,0 +1,12 @@
+features for productivity traking of students:-
```

Comment:
It's an awesome list of features to keep our design simple we can keep only one feature. Users can create goals and daily he can track a goal.